.github/workflows/release_dataops.yml

```diff
@@ -0,0 +1,56 @@
+name: Build & Publish jupyterlab-dataops Image
```

Comment:
I think we can keep one `release.yml` file and just add the multiple tags for him to watch. What do you think?

Reply:
but how can we define context (directory which should be built) based on tag?

Reply:
It's already defined in the lower part by extracting the directory name from the tag itself